src/scripts/main.js

```diff
@@ -1 +1,90 @@
 'use strict';
+
+function showNotification(posTop, posRight, description, type) {
+  const notification = document.createElement('div');
+
+  notification.dataset.qa = 'notification';
+
+  notification.style.position = 'fixed';
+
+  notification.style.margin = '0';
+  notification.style.width = '300px';
+  notification.style.padding = '10px';
+  notification.style.borderRadius = '10px';
+
+  notification.classList.add(type);
+  notification.textContent = description;
+
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  document.body.append(notification);
+
+  setTimeout(() => (notification.style.display = 'none'), 2000);
+}
+
+const firstPromise = new Promise((resolve, reject) => {
+  const timeout = setTimeout(reject, 3_000, 'First promise was rejected');
```

Comment:
The first promise will be rejected after 3 seconds if the user doesn't click. Ensure this behavior is intended, as it might not give users enough time to interact.